### ISSUE TITLE
must `tmate` package be recompiled?

### ISSUE THREAD

**Issue by alvfig** [termux/termux-packages#3521]
`tmate` is failing to execute as seen bellow. I wonder if this package must be recompiled in the official repository to back to work again. I think the `LD_LIBRARY_PATH` environment variable is correct. Android version is 6.
```
$ tmate
CANNOT LINK EXECUTABLE: cannot locate symbol "nl_langinfo" referenced by "tmate"...
page record for 0xb6f5c02c was not found (block_size=16)
$ echo $LD_LIBRARY_PATH
/data/data/com.termux/files/usr/lib
$ 
```

**Comment by fornwall** [comment]
Thanks for reporting! This should now have been fixed in version `2.2.1-2` of the tmate package - update with `pkg up`.

https://github.com/termux/termux-packages/pull/3523

**Comment by alvfig** [comment]
It worked now, thank You!
